indra/assemblers/indranet/net.py

```diff
@@ -128,6 +145,12 @@ def to_signed_graph(self, sign_dict):
             as positive edges and Inhibition and DecreaseAmount are added as
             negative edges, but a user can pass any other Statement types in
             a dictionary.
+        flattening_method : str|function
```

Comment:
Give examples of what the valid string values are.

Reply:
Fixed in https://github.com/sorgerlab/indra/pull/944/commits/c192fa3f6427261c6a7adec3074dd5e609a9a12a